packages/express/src/index.ts

```diff
@@ -1,14 +1,18 @@
 import { ExpressLogger } from './express';
-import { Options } from './types';
+import { Options, HttpLogEntry } from './types';
 
-export { ExpressLogger, Options };
-
-export function expressLogger(options?: Options): ExpressLogger {
-  return new ExpressLogger(options);
-}
+export { ExpressLogger, Options, HttpLogEntry };
 
 declare module '@debugr/core' {
-  export interface Plugins {
-    express: ExpressLogger;
+  export interface TContextFixed {
+    restRoute?: string;
+    restMethod?: string;
+  }
```

Comment:
what is this? & why `restRoute` & `restMethod`? express samotnej o žádným RESTu pokud vím nic neví, a tohle je plugin pro _express_, ne express s REST nadstavbou, tohle sem imo nepatří..

Reply:
express jde pouzit uplne bez RESTu? O_O

Reply:
samozrejme